### PR TITLE
Fix: pwsh scripts folder not there

### PR DESCRIPTION
**Pull request by cjfullerton**
Host hpcflow, matflow folders not created so users making an install on a fresh system encountered an error.

### CODE REVIEW

File: src/install-hpcflow.ps1 [hunk 1]
```diff
@@ -96,6 +98,14 @@ function Get-InstallDir {
 	return $WindowsInstallDir
 }
 
+function Check-InstallDir {
+
+	if(-Not (Test-Path $Folder)) {
+		tNew-Item -ItemType Directory $Folder
```
Comment (aplowman) [inline src/install-hpcflow.ps1:104]: Typo!

File: src/install-matflow.ps1 [hunk 2]
```diff
@@ -95,6 +97,14 @@ function Get-InstallDir {
 	return $WindowsInstallDir
 }
 
+function Check-InstallDir {
+
+	if(-Not (Test-Path $Folder)) {
+		tNew-Item -ItemType Directory $Folder
```
Comment (aplowman) [inline src/install-matflow.ps1:103]: Typo!

**Comment by aplowman** [comment]
I tried running this in a [Windows Sandbox](https://learn.microsoft.com/en-us/windows/security/application-security/application-isolation/windows-sandbox/windows-sandbox-overview) (because it's always a "clean machine"), and found that I needed to:

- Add `-UseBasicParsing` to all of the `Invoke-WebRequest` commands, this is something to do with this command using some component of Internet Explorer
- Add `-Force` to the command `New-Item -Path $profile -Type File` which is used to create the profile file if it doesn't exist. In this case, this is necessary because one of the parent directories didn't exist.

I suggest we add these changes too. It is possible that someone might try to install on Windows Server for which these problems could feasibly exist.

**Comment by cjfullerton** [comment]
Have made the following changes:

- fixed typos
- Added `-UseBasicParsing` to `Invoke-WebRequest` calls
- Added `-Force` to `New-Item` calls (this should create parent directories where they don't exist).

@aplowman Feel free to test in Windows sandbox again when you have a moment.

**Comment by cjfullerton** [comment]
Merging as this version of the script runs on the integration test workflow.

**Pull request merged**